### ISSUE TITLE
Updates to Anaconda packages

### ISSUE THREAD

**Issue by laytonjb** [ContinuumIO/anaconda-issues#9214]
### Packages Request

I'm using Conda 5.0.1 with Python 3.6.5 (just updated on 4/20/2018) on Windows 10. The version of TensorFlow is rather old (1.2). Could that be upgraded to 1.7 or perhaps 1.4? Also, it doesn't have GPU support - could that be added?

Next - using Keras to plot the model doesn't work because of the packages pydot and graphviz on Windows 10. Could they be upgraded and tested with Keras. Here is a simple code for testing:

from keras.models import Sequential
from keras.layers import Dense
from keras.utils.vis_utils import plot_model
model = Sequential()
model.add(Dense(2, input_dim=1, activation='relu'))
model.add(Dense(1, activation='sigmoid'))
plot_model(model, to_file='model_plot.png', show_shapes=True, show_layer_names=True)

(This is from https://machinelearningmastery.com/visualize-deep-learning-neural-network-model-keras/ ).

##### `conda info`

```

```

##### `conda list --show-channel-urls`

```

```
